src/utils/envFindConfigParam.cpp

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright - See the COPYRIGHT that is included with this distribution.
+ * pvAccessCPP is distributed subject to a Software License Agreement found
+ * in file LICENSE that is included with this distribution.
+ */
+
+#include <string.h>
+#include "envDefs.h"
+#include "envFindConfigParam.h"
+
+using namespace std;
```

Comment:
Is this necessary?  No c++ headers are being included.

Reply:
No, I added that while looking into build error that turned out to be missing extern "C".